kolibri/core/discovery/serializers.py

```diff
@@ -64,6 +65,11 @@ def validate(self, data):
         data.update(info)
         return super(NetworkLocationSerializer, self).validate(data)
 
+    def create(self, validated_data):
+        # Ensure 'location_type' is set to 'static' when creating new instances
+        validated_data["location_type"] = LocationTypes.Static
```

Comment:
Alternatively, we could enforce the serializer uses `StaticNetworkLocation`, but this is just fine.